src/language-js/printer-estree.js

```diff
@@ -3873,18 +3875,22 @@ function printMethodInternal(path, options, print) {
   return concat(parts);
 }
 
-function printJestEachTemplateLiteral(node, expressions, options) {
+function printJestEachTemplateLiteral(options, path, print) {
```

Comment:
Please change the order to `path, options, print` to be consistent with similar `print*` functions.